src/components/page-layout/examples/group-index-page.js

```diff
@@ -0,0 +1,112 @@
+              title: 'Guides'
+            },
+            '/PageLayout/guides/worldview/': {
+              parent: '/PageLayout/guides/',
+              title: 'Guides'
+            },
+            '/PageLayout/guides/worldview/border-styling/': {
+              parent: '/PageLayout/guides/worldview/',
+              title: 'Worldview'
+            }
+          },
+          navTabs: [
+            {
+              path: '/PageLayout/guides/',
+              title: 'Guides',
+              id: 'guides',
+              pages: [
+                {
+                  path: '/PageLayout/guides/annotations/',
+                  title: 'Annotations',
+                  id: 'annotations'
+                },
+                {
+                  path: '/PageLayout/guides/worldview/',
+                  title: 'Worldview',
+                  id: 'worldview',
+                  subPages: [
+                    {
+                      title: 'Available worldviews',
+                      description:
+                        'Identify geographic features whose characteristics are defined differently by audiences belonging to various regional, cultural, or political groups.',
+                      path: '/PageLayout/guides/worldview/avilable-worldviews/',
+                      groupOrder: 1,
+                      id: 'sp-one'
+                    },
+                    {
+                      title: 'Border styling',
+                      description:
+                        'When creating a map style with a tileset that uses worldviews, you must apply a filter to any layers that include a worldview data field.',
+                      path: '/PageLayout/guides/worldview/border-styling/',
+                      groupOrder: 2,
+                      id: 'sp-two'
+                    },
+                    {
+                      title: 'Data sources',
+                      description:
+                        'Some Mapbox vector tilesets, including Mapbox Streets v8 and Boundaries v3, include the worldview data field in several layers.',
+                      path: '/PageLayout/guides/worldview/data-sources/',
+                      groupOrder: 3,
+                      id: 'sp-three'
+                    }
+                  ]
+                },
+                {
+                  path: '/PageLayout/guides/expressions/',
+                  title: 'Expressions',
+                  id: 'expressions'
+                }
+              ]
+            },
+            {
+              path: '/PageLayout/specification/',
+              title: 'Specification',
+              id: 'specification'
+            },
+            {
+              path: '/PageLayout/examples/',
+              title: 'Examples',
+              id: 'examples'
+            },
+            {
+              path: '/PageLayout/demo/',
+              title: 'Demo',
+              id: 'demo'
+            }
+          ]
+        }}
+      ></PageLayout>
```

Comment:
I think we can make PageLayout self closing:

```suggestion
      />
```